Create ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md (#2699)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-14-0-as-a-developer-i-want-to-implement-a-generic-datapoints-export.md

---
date: ""

> title: New datapoint export selector

Reviewer: ```suggestion
title: New datapoint export component
```

> title: As a developer, I want to implement a generic datapoints export component (#7129) [GRAFT][release/cd]  (#7721)

Author: The Claude AI suggested to use the following title:
```suggestion
title: New datapoint export selector
```

Reviewer 2: In our repo this component is called: `datapoints-export-selector` so I would go with:
```suggestion
title: Datapoints export selector
```

or

```suggestion
title: Datapoints export selector component
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60214
version: 1021.14.0
---

> As a developer, I want to implement a generic datapoints export component (#7129) [GRAFT][release/cd]  (#7721)

Author: The Claude AI suggested to use the following body:
```suggestion
To enable users to easily export datapoints from Cumulocity IoT, a new generic datapoints export component has been implemented. This component allows users to select the devices, the time range and the format for the datapoints export. The exported file will then be generated asynchronously and can be downloaded by the user once it is ready. This new component makes it much simpler for users to get the device data they need out of Cumulocity IoT in their desired format.
```

Reviewer 2: ```suggestion
To allow users to easily export datapoints from Cumulocity IoT, a new generic datapoint export component has been implemented. This component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoints export. Depending on the size of the datapoints records to be exported, the exported file is then generated and downloaded directly or emailed when ready.  This new component makes it much simpler for users to get the device data they need out of Cumulocity IoT in their desired format.
```
or without the last sentence which seems unnecessary:

```suggestion
To allow users to easily export datapoints from Cumulocity IoT, a new generic datapoint export component has been implemented. This component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoints export. Depending on the size of the datapoints records to be exported, the exported file is then generated and downloaded directly or emailed when ready.
```

> To enable users to easily export datapoints from {{< product-c8y-iot >}}, a new generic datapoint export component has been implemented. This component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoint export. Depending on the size of the datapoint records to be exported, the exported file is then generated and downloaded directly or emailed when ready. This new component makes it much simpler for users to get the device data they need out of {{< product-c8y-iot >}} in their desired format.

Reviewer: ```suggestion
To enable users to easily export datapoints from {{< product-c8y-iot >}}, a new generic datapoint export selector has been implemented. This new component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoint export. Depending on the size of the datapoint records to be exported, the exported file is then generated and downloaded directly or emailed when ready. The datapoint export selector component makes it much simpler for users to get the device data they need out of {{< product-c8y-iot >}} in their desired format.
```

> To enable users to easily export datapoints from {{< product-c8y-iot >}}, a new generic datapoint export selector has been implemented. This new component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoint export. Depending on the size of the datapoint records to be exported, the exported file is then generated and downloaded directly or emailed when ready. The datapoint export selector component makes it much simpler for users to get the device data they need out of {{< product-c8y-iot >}} in their desired format.

Reviewer: ```suggestion
To enable users to easily export datapoints from {{< product-c8y-iot >}}, a new generic datapoint export selector has been implemented. This new component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoint export. Depending on the size of the datapoint records to be exported, the exported file is then generated and downloaded directly or emailed when ready. The datapoint export component makes it much simpler for users to get the device data they need out of {{< product-c8y-iot >}} in their desired format.
```

> To allow users to easily export datapoints from Cumulocity IoT, a new generic datapoint export component has been implemented. This component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoints export. Depending on the size of the datapoints records to be exported, the exported file is then generated and downloaded directly or emailed when ready.  This new component makes it much simpler for users to get the device data they need out of Cumulocity IoT in their desired format.

Reviewer: ```suggestion
To enable users to easily export datapoints from {{< product-c8y-iot >}}, a new generic datapoint export component has been implemented. This component allows users to select the time range, export type, aggregation (based on the export type) and file format for the datapoint export. Depending on the size of the datapoint records to be exported, the exported file is then generated and downloaded directly or emailed when ready. This new component makes it much simpler for users to get the device data they need out of {{< product-c8y-iot >}} in their desired format.
```
